### ISSUE TITLE
Picasso library need to be updated

### ISSUE THREAD

**Issue by Renascienza** [mercadopago/px-android#1925]
Picasso dependency needs to be updated, due public API has changed. 

If a project have current Picasso version (2.7.x) you have this error in final of checkout operation:

java.lang.NoSuchMethodError: No static method with(Landroid/content/Context;)Lcom/squareup/picasso/Picasso; in class Lcom/squareup/picasso/Picasso;

**Comment by cgaggino** [comment]
Hi Renascienza, thanks for contacting us.

We have no plans in the mid term to update Picasso's dependency. Square says that Picasso 2.7+ is not backward compatible. Therefore, if we update it, we will break other projects which are already implementing us.

Kind regards.

**Comment by Renascienza** [comment]
Hi, cgaggino.
This is exactly the problem.

My project have another libraries that uses Picasso 2.7+ (and I can't avoid it). 

So, gradle use this version instead of the old, and give me a stupid "No Such Method Error" (because the library try to load a icon and fail) prevent the conclusion of the trail (the money is transfered, but the flow doesn't have a result, so I can't have the confirmation in onActivityResult to send the order to my server. Bad, right?)

So, if you can suggest a workaround about this, would be appreciated. If you don't what your suggestion? Just give up to use MercadoPago as my paying system for good and try to find some competitor?

Needless to say, from my IT experience (some), there is no such thing as "our library can't have the dependences updated". If something can break if you update, you fix this possibility on code. But is not my team so... 

Even so, I have a problem that may affect other library users and deserve to be fixed, right?

I was also a little dissatisfied with how the problem was handled. If someone open an issue, is because that person have a problem. Just to say "_We are not going to do anything, we don't care, Issue closed, kind regards_ (in some cases, sound like a polite expression to "get lost")" doesn't seems to be a correct way to deal with. Looks like a bad attitude...

Anyway, I will open a more specific issue with the exact error found.

**Comment by cgaggino** [comment]
Hi Renascienza,

I apologies if i my answer was not satisfactory, and I will try to explain myself

The issue was related to update Picasso and that is something we can't do, because like you, there are many other people using this library which do not have compatibility problems with Picasso and if we update Picasso, a lot of people would start to have problems

The same problem was reported a while ago: https://github.com/mercadopago/px-android/pull/1547

On the other hand, I took a look to the compatibilities issues and I think I found a workaround that could help you. I will come with news later today in your new issue.

Kind regards.
